src/helm/config/tokenizer_configs.yaml

```diff
@@ -83,6 +83,13 @@ tokenizer_configs:
     end_of_text_token: ""
     prefix_token: ":"
 
+  # Databricks
+  - name: databricks/dbrx-instruct
+    tokenizer_spec:
+      class_name: "helm.tokenizers.huggingface_tokenizer.HuggingFaceTokenizer"
+    end_of_text_token: "<|endoftext|>"
+    prefix_token: "<|endoftext|>"
```

Comment:
should be "<|begin_of_text|>"

Reply:
Actually `"<|endoftext|>"` is correct (it's common for the bos and eos tokens to be the same)
```
from transformers import AutoTokenizer
tokenizer = AutoTokenizer.from_pretrained("databricks/dbrx-instruct")
print(tokenizer.bos_token)  # prints <|endoftext|>
```

Reply:
Ah i see, got it